app/instrumentation-tests/src/org/commcare/utils/CustomMatchers.kt

```diff
@@ -116,7 +117,11 @@ object CustomMatchers {
                     return false
                 }
                 val pixels = target.textSize
-                val actualSize = pixels / target.getResources().displayMetrics.scaledDensity
+
+                val actualSize = (pixels / target.resources.displayMetrics.scaledDensity).roundToInt()
+                Logger.log(CustomMatchers::class.simpleName + ".withFontSize()", "View text size: $pixels")
+                Logger.log(CustomMatchers::class.simpleName + ".withFontSize()", "Actual size: $actualSize")
+                Logger.log(CustomMatchers::class.simpleName + ".withFontSize()", "Font scale: " + target.resources.configuration.fontScale)
```

Comment:
do we still need the logs ? (assuming they were for debugging)

Reply:
+1 on this comment

Reply:
Not needed, the plan was to remove them - [62b251f](https://github.com/dimagi/commcare-android/pull/3403/commits/62b251f9a00c885643060612478ab7db8ed1e047)